### PR TITLE
add warpdrive.sh to transparently invoke the container instance of hyperspace.py

### LINKED ISSUE #51: Enable single command invocation of hyperspace from outside of the container
Otherwise testing gets annoying

### PR DESCRIPTION
**Pull request by theyoyojo**
now leaving the local galactic cluster (fixes #51)

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 srvname = 'kdlp.underground.software'
 production = False
 
+use_podman = False
```
Comment (charliemirabile) [inline config.py:8]: I think doing this via config is overkill, just allow overriding in invocation of hyperspace by setting an env var, user can make an alias if they want

File: warpdrive.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+#
+# the warp drive allows you to enter hyperspace
+
+cd "$(dirname "$0")"
+
+DOCKER="docker"
```
Comment (charliemirabile) [inline warpdrive.sh:7]: prefer `: ${DOCKER:=sudo docker}`, and then folks can just run with `DOCKER=podman ./hyperspace ...` if they don't use docker

File: warpdrive.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+USE_PODMAN=$(./config.py use_podman)
+
+if [ "$USE_PODMAN" == "True" ]; then
+	DOCKER="podman"
+	echo "using podman instead of docker"
+fi
```
Comment (charliemirabile) [inline warpdrive.sh:13]: then all this can go

File: warpdrive.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+cat <<EOF | sudo $DOCKER exec -i docker_orbit_1 /bin/sh
```
Comment (charliemirabile) [inline warpdrive.sh:15]: and then just drop sudo here. With podman, you don't nessecarily want sudo

**Review by charliemirabile — CHANGES_REQUESTED** [review]
nit about choosing between docker and podman

File: warpdrive.sh [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+source /radius-venv/bin/activate
```
Comment (charliemirabile) [inline warpdrive.sh:16]: the `.` command is actually more portable/correct than `source`

File: config.py (hunk 1 above)
Comment (theyoyojo) [reply config.py:8]: I was thinking we could use this for some other small utilities but I can always add it in later

File: warpdrive.sh (hunk 2 above)
Comment (theyoyojo) [reply warpdrive.sh:7]: Is that basically `DOCKER=${DOCKER:-sudo docker}`?

File: warpdrive.sh (hunk 3 above)
Comment (theyoyojo) [reply warpdrive.sh:13]: yeah

File: warpdrive.sh (hunk 4 above)
Comment (theyoyojo) [reply warpdrive.sh:15]: true

File: warpdrive.sh (hunk 5 above)
Comment (theyoyojo) [reply warpdrive.sh:16]: and shorter

**Review by charliemirabile — APPROVED** [review]
lgtm

**Pull request merged**